src/LoriotAzureFunctions/LoriotAzureFunctions.csproj~5d6acc3... merges seperated function projects to one single project

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The excess files starting with LoriotAzureFunctions.csproj~ should not be committed.